db/seeds.rb

```diff
@@ -22,3 +22,34 @@
 
   user.save!
 end
+
+skill = Skill.new(name: "Javascript")
+skill.save!
+
+skill = Skill.new(name: "Ruby")
+skill.save!
+
+skill = Skill.new(name: "CSS")
+skill.save!
+
+skill = Skill.new(name: "React")
+skill.save!
+
+role = Role.new(name: "Frontend")
+role.save!
+
+role = Role.new(name: "Backend")
+role.save!
+
+role = Role.new(name: "freelance")
+role.save!
+
+role = Role.new(name: "Full-stack")
+role.save!
+
+post = Post.new(skill.all.sample)
```

Comment:
The post as it stands will not create, please check the schema to see what items are required to make a post.